Tests/Marketplace/upload_packs.py

```diff
@@ -158,10 +159,28 @@ def update_index_folder(index_folder_path, pack_name, pack_path):
         index_folder_subdirectories = [d for d in os.listdir(index_folder_path) if
                                        os.path.isdir(os.path.join(index_folder_path, d))]
         index_pack_path = os.path.join(index_folder_path, pack_name)
+        metadata_files_in_index = glob.glob(f"{index_pack_path}/**/metadata-*.json")
```

Comment:
```suggestion
        metadata_files_in_index = glob.glob(f"{index_pack_path}/**/metadata-*.json", recursive=True)
```
Aiiiiii, `index_pack_path` already points to pack directory inside `index` folder (you have added additional subfolder) and by default the `recursive` flag is set to False. 
Or alternatively just remove subfolder:  `glob.glob(f"{index_pack_path}/metadata-*.json")`